src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -1,54 +1,110 @@
 
+    const [searchAdvancedFilters = {}] = useOnyx(ONYXKEYS.FORMS.SEARCH_ADVANCED_FILTERS_FORM);
+
     const advancedFilters = useMemo(
         () => [
             {
-                title: getFilterDisplayTitle({}, 'title'),
+                title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_ROOT_KEYS.TYPE, translate),
                 description: 'common.type' as const,
                 route: ROUTES.SEARCH_ADVANCED_FILTERS_TYPE,
             },
             {
-                title: getFilterDisplayTitle({}, 'date'),
+                title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_ROOT_KEYS.STATUS, translate),
+                description: 'search.filters.status' as const,
+                route: ROUTES.SEARCH_ADVANCED_FILTERS_STATUS,
+            },
+            {
+                title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_FILTER_KEYS.DATE, translate),
                 description: 'common.date' as const,
                 route: ROUTES.SEARCH_ADVANCED_FILTERS_DATE,
             },
         ],
-        [],
+        [searchAdvancedFilters, translate],
     );
 
+    const onFormSubmit = () => {
+        const query = SearchUtils.buildQueryStringFromFilters(searchAdvancedFilters);
+        SearchActions.clearAdvancedFilters();
+        Navigation.navigate(
+            ROUTES.SEARCH_CENTRAL_PANE.getRoute({
+                query,
+                isCustomQuery: true,
+            }),
+        );
+    };
+
     return (
-        <View>
-            {advancedFilters.map((item) => {
-                const onPress = singleExecution(waitForNavigate(() => Navigation.navigate(item.route)));
-
-                return (
-                    <MenuItemWithTopDescription
-                        key={item.description}
-                        title={item.title}
-                        description={translate(item.description)}
-                        shouldShowRightIcon
-                        onPress={onPress}
-                    />
-                );
-            })}
+        <View style={[styles.flex1, styles.justifyContentBetween]}>
+            <View>
+                {advancedFilters.map((item) => {
+                    const onPress = singleExecution(waitForNavigate(() => Navigation.navigate(item.route)));
+
+                    return (
+                        <MenuItemWithTopDescription
+                            key={item.description}
+                            title={item.title}
+                            description={translate(item.description)}
+                            shouldShowRightIcon
+                            onPress={onPress}
+                        />
+                    );
+                })}
+            </View>
+            <FormAlertWithSubmitButton
+                buttonText={translate('search.viewResults')}
+                containerStyles={[styles.m4]}
+                onSubmit={onFormSubmit}
+            />
```

Comment:
I think we can keep it as is for now so that we don't block other issues, but I think we should disable it offline because the Search API command only works while the user is online.